selfdrive/camerad/cameras/camera_qcom.cc

```diff
@@ -1732,7 +1734,9 @@ void actuator_move(CameraState *s, uint16_t target) {
   //LOGD("actuator move focus: %d", err);
 
   s->cur_step_pos = dest_step_pos;
+  pthread_mutex_lock(&s->frame_info_lock);
   s->cur_lens_pos = actuator_cfg_data.cfg.move.curr_lens_pos;
```

Comment:
is this necessary?

Reply:
it looks like that this value will be written in ops_thread(by calling do_autofocus) and read in the main thread.did I get that right?

Reply:
the one constantly being written is called true_len_pos? I think the cur_lens_pos should be just a read-only number returned from the device.